src/auxiliary/math.jl

```diff
@@ -5,6 +5,101 @@
 @muladd begin
 #! format: noindent
 
+const TRIXI_UUID = UUID("a7f1ee26-1774-49b1-8366-f1abc58fbfcb")
+
+"""
+    set_sqrt_type(type; force = true)
```

Comment:
```suggestion
    Trixi.set_sqrt_type(type; force = true)
```
I think we shouldn't `export` this function.